index.js

```diff
@@ -87,10 +87,13 @@ function wrappedSpawnFunction (fn, workingDir) {
       cmdi = options.args.indexOf('-c')
       if (cmdi !== -1) {
         c = options.args[cmdi + 1]
-        re = /^\s*((?:[^\= ]*\=[^\=\s]*\s*)*)([^\s]+)/
+        re = /^\s*((?:[^\= ]*\=[^\=\s]*\s*)*)([^\s]+|"[^"]+"|'[^']+')/
         match = c.match(re)
         if (match) {
           exe = path.basename(match[2])
+          // Strip trailing quotes from the basename.
+          exe = exe.replace(/['"]*$/, '');
```

Comment:
This should be only done conditionally if `match[2]` starts with the quote in question, and only one char stripped.

Something like this?

```javascript
var quote = match[2].charAt(0)
if ((quote === '"' || quote === '\'') && quote === exe.slice(-1)) {
  exe = exe.slice(0, -1)
}
```